packages/vhd-lib/src/Vhd/VhdFile.js

```diff
@@ -458,4 +450,28 @@ export default class Vhd {
       header.parentLocatorEntry[i].platformDataOffset = 0
     }
   }
+
+  readParentLocatorData(parentLocatorId) {
+    assert(parentLocatorId >= 0)
+    assert(parentLocatorId < 8)
+    assert.notStrictEqual(this.header, undefined)
+    const { platformDataOffset, platformDataSpace } = this.header.parentLocatorEntry[parentLocatorId]
+    if (platformDataSpace === 0) {
+      return
+    }
+    return this._read(platformDataOffset, platformDataSpace)
+  }
+
+  writeParentLocator(parentLocatorId, data) {
+    assert(parentLocatorId >= 0)
+    assert(parentLocatorId < 8)
+    assert.notStrictEqual(this.header, undefined)
+
+    const { platformDataOffset, platformDataSpace } = this.header.parentLocatorEntry[parentLocatorId]
+    if (platformDataSpace === 0) {
+      return
+    }
```

Comment:
I don't understand this, shouldn't we throw an error?

Reply:
I think we should throw an error only if data is not empty